docs/kns_apps_tutorial.md

```diff
@@ -0,0 +1,347 @@
+# Jarvice KNS Applications tutorial
+
+![GlobalProcess](img/apps_tutorial/SuperNimbix.svg)
+
+Note: it is assumed that the user is already familliar with Jarvice Push to Compute (P2C). Please refer to [apps tutorial](apps_tutorial.md).
```

Comment:
We are not saying anything regarding that the user should use a region / cluster / zone in which KNS is installed and that information about this should be asked to the local System Administrator.

Reply:
You mean select the proper zone inside the Jarvice Portal ?
In theory, its part of the general training of Jarvice.

Reply:
OK. Nothing to do then.